### PR TITLE
Add new idea langauge.

### PR DESCRIPTION
**Pull request by p-buda-robo**

### CODE REVIEW

**Comment by p-buda-robo** [comment]
@p-buda I added the requested changes. Feel free to merge!

File: robo-idea.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+my idea:
+Lipsum.
```
Comment (p-buda) [inline robo-idea.txt]: Perhaps we need more meat?

Comment (p-buda-robo) [reply robo-idea.txt]: Done!

**Comment by p-buda** [comment]
Merging.

**Comment by p-buda** [comment]
Done

**Pull request closed without merging**